src/lua/LuaEngine.cpp

```diff
@@ -775,6 +775,39 @@ static int l_engine_set_music_volume(lua_State *l)
 	return 0;
 }
 
+static int l_engine_get_mute_on_pause(lua_State *l)
+{
+	lua_pushboolean(l, Pi::config->Int("AudioMuteOnPause") != 0);
+	return 1;
+}
+
+static int l_engine_set_mute_on_pause(lua_State *l)
+{
+	if (lua_isnone(l, 1)) {
+		return luaL_error(l, "SetMuteOnPause takes one boolean argument");
+	}
+	const bool mute= lua_toboolean(l, 1);
```

Comment:
Nitpick: the codebase's stylistic conventions require a space on both sides of the equals-sign here. We used to have a clang-format nag running on PRs to catch this stuff automatically, but it got far too opinionated without config-file levers to disable its opinions.